app/js/shared/ucBackground/ucBackground.directive.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
The contents should be wrapped in an anonymous function to protect it from namespace collisions and minification. See the styleguide: https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#iife

All the .js files have this so you can just copy paste the first and last lines from any .js file inside the app directory